app/controllers/guides_controller.rb

```diff
@@ -71,31 +71,28 @@ def confirm_unpublish
 
 private
 
-  def manage!(guide, action, opts = {})
-    message = opts.fetch(:message, nil)
-
-    guide_manager = GuideManager.new(guide: guide, user: current_user)
-    guide_manager.public_send("#{action}!")
-
-    redirect_to edit_guide_path(guide), notice: message
-  end
-
   def manage(guide, action, opts = {})
     redirect = opts.fetch(:redirect, edit_guide_path(guide))
     message = opts.fetch(:message, nil)
 
+    @guide_form = GuideForm.build(
+      guide: guide,
+      edition: guide.latest_edition,
+      user: current_user
+    )
+
+    if guide_has_changed_since_editing?(guide)
+      flash.now[:error] = "The guide has changed since you started editing"
```

Comment:
The problem is that what they do is going to depend a bit on the type of change they've made.. They're in conflict resolution land. They might want to merge the changes with 'theirs' or 'mine' as a base.

We could talk about navigating back to the guides index, reopening the guide they're editing, comparing the differences, and leaving a question mark over how to proceed. Sounds like a bit of a long flash message :/ Could be done though.